code/game/area/areas/ruins/jungle.dm

```diff
@@ -130,3 +130,45 @@
 /area/ruin/jungle/syndifort/jerry
 	name = "Syndicate Fort Tower"
 	icon_state = "bridge"
+
+// Forgotten Botanical Research Facility
+
+/area/ruin/jungle/botanicalresearch
+	name = "Forgotten Botanical Research Facility"
+	icon_state = "red"
+
+/area/ruin/jungle/botanicalresearch/command
+	name = "Forgotten Botanical Research Facility Administration"
+	icon_state = "blue"
+
+/area/ruin/jungle/botanicalresearch/cleanroom
+	name = "Forgotten Botanical Research Facility Cleanroom"
+	icon_state = "yellow"
+
+/area/ruin/jungle/botanicalresearch/crew
+	name = "Forgotten Botanical Research Facility Living Quarters"
+	icon_state = "crew_quarters"
+
+/area/ruin/jungle/botanicalresearch/engineering
+	name = "Forgotten Botanical Research Facility Engineering"
+	icon_state = "engine"
+
+/area/ruin/jungle/botanicalresearch/vault
+	name = "Forgotten Botanical Research Facility Vault"
+	icon_state = "storage"
+
+/area/ruin/jungle/botanicalresearch/checkpoint
+	name = "Forgotten Botanical Research Facility Security Checkpoint"
+	icon_state = "security"
+
+/area/ruin/jungle/botanicalresearch/kitchen
+	name = "Forgotten Botanical Research Facility Kitchen"
+	icon_state = "yellow"
```

Comment:
each area is meant for a part of the facility that needs power and for the kitchen it specifically did not feel right to make it part of the living quarters area

Reply:
i will not be willing to follow through with this request primarily because each area is defined for a specific reason:

crew quarters: far from administration
kitchen: only connected to quarters by like 3 walls
cleanroom: literally a mile away from the other wings bar admin
vault: keeping it powered
engi: it's engi
command: yea

Reply:
If you are not willing to follow through then we will not be merging the pr.

Reply:
alright, i've reduced areas to pretty much as much as i can without causing APC foolery or otherwise poorly done areas, i've put the cleanroom area as the base area

Reply:
Righto, if I see any other changes to them that can be done, I'll note it in my review. Expect one in the next few

Reply:
done